model/src/main/scala/uk/gov/ons/addressIndex/model/server/response/AddressResponse.scala

```diff
@@ -185,7 +185,9 @@ object AddressResponseAddress {
     val geo: Try[AddressResponseGeo] = for {
```

Comment:
They still need to be created in some external files. I did the prefix directly in the name to avoid naming collisions, but we can just use package prefix every time we are using them.